src/App.js

```diff
@@ -13,6 +16,7 @@ function App() {
   const [productsCategories, setProductsCategories] = useState([]);
   const [featuredBanners, setFeaturedBanners] = useState([]);
   const [featuredProducts, setFeaturedProducts] = useState([]);
+  const [atHomePage, setAtHomePage] = useState(true);
```

Comment:
Good job 👍, you understood the concept of `conditional rendering`, later this will be replaced for the react router.